node/src/header_ex.rs

```diff
@@ -33,14 +34,21 @@ use crate::utils::{protocol_id, OneshotResultSender};
 
 /// Max request size in bytes
 const REQUEST_SIZE_MAXIMUM: usize = 1024;
+/// Time limit on reading/writing a request
+const REQUEST_TIME_LIMIT: Duration = Duration::from_secs(1);
 /// Max response size in bytes
 const RESPONSE_SIZE_MAXIMUM: usize = 10 * 1024 * 1024;
+/// Time limit on reading/writing a response
```

Comment:
can we align names for the size, like `REQUEST_SIZE_LIMIT`?

Reply:
done